examples/with-typescript-ionic/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "with-typescript-ionic",
```

Comment:
Let's rename the example to `with-ionic-typescript`

Reply:
Renamed it my latest commit 👍